src/users/services/users.service.ts

```diff
@@ -45,4 +45,10 @@ export class UsersService {
       .get<any>(`/api/user-details?userId=${userId}`)
       .pipe(catchError((error: any) => throwError(error.json())));
   }
+
+  public retrieveAccessType(organisationProfileIds: Array<string>): Observable<any> {
```

Comment:
expect the return type to be strongly typed too. Array should be written as `string[]`.
```suggestion
  public retrieveAccessType(organisationProfileIds: string[]): Observable< JurisdictionAccessType[]> {
```